content/departments/product/design/design-and-engineering-collaboration.md

```diff
@@ -0,0 +1,39 @@
+# How design and engineering collaborate
+
+## We make time for design before engineering begins
+
+By beginning with design we help the entire team explore product requirements in the least expensive and most collaborative medium. When we all make time to collaborate in the design stage, we provide everyone with the ability to review the solution before development. This process should lead to stronger outcomes with less technical debt, as engineering’s systems design has a better chance of being correct on the first try.
+
+## We design together, throughout the project
+
+From planning to QA, everyone shapes the outcome of the product, each step involves some measure of design, product, and engineering input. When we have feedback from a wide range of disciplines, we produce a much better product.
+
+As a project proceeds, design and engineering collaborate to modify the effort as required by various constraints. Both parties are flexible and seek to understand tradeoffs to arrive at the best solution given available resources. We understand that at times both design and tech debt is generated during this process. We track this debt and seek to reduce it when time allows.
```

Comment:
Tiny nitpick: most of the time in this doc, Design and Engineering are capitalized when used as descriptions of the disciplines, but not here.

Reply:
Ah, jeez, that was a bit of a shitshow. I went through and capitalized the departments while NOT capitalizing references to the practices. I hope I got it all right. 

Thanks for your diligence.